tests/mock_dynamixel.py

```diff
@@ -0,0 +1,84 @@
+"""Mocked classes and functions from dynamixel_sdk to allow for continuous integration
```

Comment:
Nit: A general comment here. I am not sure we need to have a mock of every type of motor. For example, instead of having a `mock_dynamixel`, `mock_feetech`, etc and having to add a new mock motor class every time we include a new motor in our codebase, we may just want to use one class `mock_motor` that will be used for any motor when we need to test code but don't have actual motors connected. I believe this should also apply to the cameras and we should just have one `mock_camera` class. What do you think?

Reply:
The benefit of mocking each camera sdk and motor sdk is that we test more code logic.
With only a MockMotorsBus class, if we add a bug in the DynamixelMotorsBus, we won't be able to capture it.

Reply:
After discussing with Simon, we might want to come up with a new design for testing hardware which simplifies mocking.
However, this might require refactoring of the current robot devices.
We will first merge this test PR and redesign test after the refactor.

Reply:
ok gotcha!